build.gradle.kts

```diff
@@ -68,7 +68,7 @@ tasks.test {
 }
 
 tasks.withType<KotlinCompile> {
-    kotlinOptions.jvmTarget = "1.8"
+    kotlinOptions.jvmTarget = "11"
```

Comment:
Are you sure about this? I don't know what it should be, but the Ci deployment is compiling with java-version 17

Reply:
@tcasstevens - should we update this to 17?

Reply:
@btmonier If it compiles and works correctly I would say yes